documentation/business-analyst-activities.md

```diff
@@ -1 +1,17 @@
 # Business Analyst Activities
+
+**How much detail should we go into with regards to setting up GITHUB account?
+
+
+## Updating Polygon Mappings in the mapped_locations.yml file
+This section details the process for updating the search mappings so that a polygon search is used in place of a point search.
+
+1. Navigate to the "Code" tab within github and press "go to file"
+2. Search for "mapped_locations.yml" and click on the file.
+  * This file contains a list of all of the mappings that have been applied, in alphabetical order.
+3. Navigate to the desired row, and input your new mapping in the following format:
+  * -['location that is searched by user', 'name of polygon we want to return']
```

Comment:
The polygon names can be found here: https://github.com/DFE-Digital/teaching-vacancies/tree/master/lib/tasks/data

Reply:
Tip: You can surround code with backticks (`) in Markdown to format it nicely, e.g.

`-['location that is searched by user', 'name of polygon we want to return']`

Reply:
Added link